File: src/org/stormgears/utils/dsio/DummyJoystick.java
package org.stormgears.utils.dsio;

public class DummyJoystick implements IRawJoystick {
	@Override
	public double getJoystickX() {
		return 0;
	}

	@Override
	public double getJoystickY() {
		return 0;
	}

	@Override
	public double getJoystickZ() {
		return 0;
	}

	@Override
	public boolean getRawButton(int button) {
		return false;
	}
}

File: src/main/java/org/stormgears/utils/dsio/DummyJoystick.kt
package org.stormgears.utils.dsio

class DummyJoystick : IRawJoystick {
	override val joystickX: Double = 0.0

	override val joystickY: Double = 0.0

	override val joystickZ: Double = 0.0

	override val throttleV: Double = 0.0

	override fun getRawButton(button: Int): Boolean {
		return false
	}
}
